pkg/common-controller/groupsnapshot_controller_helper.go

```diff
@@ -558,9 +558,13 @@ func (ctrl *csiSnapshotCommonController) createSnapshotsForGroupSnapshotContent(
 
 		volumeSnapshotNamespace := groupSnapshotContent.Spec.VolumeGroupSnapshotRef.Namespace
 
+		label := make(map[string]string)
+		label[utils.VolumeGroupSnapshotNameLabel] = groupSnapshotContent.Spec.VolumeGroupSnapshotRef.Name
+
 		volumeSnapshotContent := &crdv1.VolumeSnapshotContent{
 			ObjectMeta: metav1.ObjectMeta{
-				Name: volumeSnapshotContentName,
+				Name:   volumeSnapshotContentName,
+				Labels: label,
```

Comment:
Why is the label set on the VolumeSnapshotContent, not on VolumeSnapshot?

Reply:
That way, we can easily understand if a VolumeSnapshotContent is a member of a VolumeGroupSnapshot.
We could use  `VolumeGroupSnapshot.status.volumeGroupSnapshotHandle` too. Let me try that approach.

Reply:
I removed the label and used `volumeGroupSnapshotHandle`